cirq/ops/phased_iswap_gate.py

```diff
@@ -0,0 +1,202 @@
+        for eigenvalue, projector in self._iswap._eigen_components():
+            new_projector = phase_matrix @ projector @ inverse_phase_matrix
+            eigen_components.append((eigenvalue, new_projector))
+        return eigen_components
+
+    def _apply_unitary_(self, args: 'protocols.ApplyUnitaryArgs'
+                       ) -> Optional[np.ndarray]:
+        if protocols.is_parameterized(self):
+            return NotImplemented
+        if self.exponent != 1:
+            return NotImplemented
+
+        phase = np.exp(1j * np.pi * self.phase_exponent)
+        zo = args.subspace_index(0b01)
+        oz = args.subspace_index(0b10)
+        args.target_tensor[zo] *= phase
+        args.target_tensor[oz] *= phase.conjugate()
+        args.target_tensor = self._iswap._apply_unitary_(args)
+        assert args.target_tensor is not None
+        args.target_tensor[zo] *= phase.conjugate()
+        args.target_tensor[oz] *= phase
+        return args.target_tensor
+
+    def _decompose_(self, qubits: Sequence[raw_types.Qid]) -> op_tree.OP_TREE:
+        if len(qubits) != 2:
+            raise ValueError(f'Expected two qubits, got {len(qubits)}')
+        a, b = qubits
+
+        yield cirq.Z(a)**self.phase_exponent
+        yield cirq.Z(b)**-self.phase_exponent
+        yield from protocols.decompose_once_with_qubits(self._iswap, qubits)
+        yield cirq.Z(a)**-self.phase_exponent
+        yield cirq.Z(b)**self.phase_exponent
+
+    def _pauli_expansion_(self) -> value.LinearDict[str]:
+        if self._is_parameterized_():
+            return NotImplemented
+        expansion = protocols.pauli_expansion(self._iswap)
+        assert set(expansion.keys()).issubset({'II', 'XX', 'YY', 'ZZ'})
+        assert np.isclose(expansion['XX'], expansion['YY'])
+
+        v = (expansion['XX'] + expansion['YY']) / 2
+        phase_angle = np.pi * self.phase_exponent
+        c, s = np.cos(2 * phase_angle), np.sin(2 * phase_angle)
+
+        return value.LinearDict({
+            'II': expansion['II'],
+            'XX': c * v,
+            'YY': c * v,
+            'XY': s * v,
+            'YX': -s * v,
+            'ZZ': expansion['ZZ'],
+        })
+
+    def __str__(self) -> str:
+        if self.exponent == 1:
+            return 'PhasedISWAP'
+        return f'PhasedISWAP**{self.exponent}'
+
+    def __repr__(self):
+        phase_exponent = proper_repr(self._phase_exponent)
+        args = [f'phase_exponent={phase_exponent}']
+        if self.exponent != 1:
+            exponent = proper_repr(self.exponent)
+            args.append(f'exponent={exponent}')
+        if self._global_shift != 0:
+            global_shift = proper_repr(self._global_shift)
+            args.append(f'global_shift={global_shift}')
+        arg_string = ', '.join(args)
+        return f'cirq.PhasedISwapPowGate({arg_string})'
+
+
+# The ISWAP gate conjugated by T⊗T^-1.
+#
+# Matrix of GivensRotation^t is
+#
+#     [[1, 0, 0, 0],
```

Comment:
How is this equation affected by the global shift?

Reply:
Well, it'd gain phase, but we've removed global shift, so this is now moot.